build/ci/.azure-devops-stages-docs.yml

```diff
@@ -1,23 +1,15 @@
 
 stages:
-- stage: Setup
```

Comment:
I don't see anything that would depend on gitversion for the docs generation.

Reply:
Since we want to share the same template (download-winui-converted-tree.yml) between docs (which doesn't need gitversion) and the main CI (which needs it), this will need some more CI refactoring. I'll revert for now, but let's discuss and do the change separately if my assumption is right.